Toolchain/BuildIt.sh

```diff
@@ -182,8 +192,9 @@ popd
 # === DOWNLOAD AND PATCH ===
 
 pushd "$DIR/Tarballs"
-    # Build aarch64-gdb for cross-debugging support on x86 systems
-    if [ "$ARCH" = "aarch64" ]; then
+    # Build gdb for cross-debugging support
+    if ! has_gdb "$ARCH"; then
```

Comment:
i feel that the value of this condition wonʼt change between the checks; care to extract it to a variable to avoid needless calls?